diff --git a/node_modules/monaco-editor/esm/vs/base/common/marked/marked.js b/node_modules/monaco-editor/esm/vs/base/common/marked/marked.js
index 545a093..5f67ffc 100644
--- a/node_modules/monaco-editor/esm/vs/base/common/marked/marked.js
+++ b/node_modules/monaco-editor/esm/vs/base/common/marked/marked.js
@@ -10,7 +10,7 @@
  */
 
 // ESM-uncomment-begin
-let __marked_exports = {};
+let __marked_exports = typeof define === 'function' && define.amd ? {} : null;
 (function() {
   function define(deps, factory) {
     factory(__marked_exports);
